### PR TITLE
Bibolton/skylight dash

### PR DESCRIPTION
**Pull request by BillyBolton**
Asana: https://app.asana.com/0/1207589688934697/1207591487804448/f

Please review this PR in tandem: https://github.com/Accedian/skylight-aaa/pull/189

### CODE REVIEW

**Review by DiodatoNocita — APPROVED** [review]

File: dashboards/Skylight-aaa.json [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
```
Comment (berupp) [inline dashboards/Skylight-aaa.json:409]: "uid": "${DS_PROMETHEUS}"

File: dashboards/Skylight-aaa.json [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+            "uid": "PBFA97CFB590B2093"
```
Comment (berupp) [inline dashboards/Skylight-aaa.json:527]: "uid": "${DS_PROMETHEUS}"

File: dashboards/Skylight-aaa.json [hunk 3]
```diff
@@ -0,0 +1,795 @@
...
```
Comment (berupp) [inline dashboards/Skylight-aaa.json:643]: "uid": "${DS_PROMETHEUS}"

**Review by berupp — APPROVED** [review]
An important test is to export a dashboard (Share -> Export -> with Export for sharing externally active)
Then import. Ideally on a different system. Use mine `https://bro4.datahub2.npav.accedian.net/`
Then you'll immediately see if you missed a datasource.

**Pull request merged**